+dj/Table.m

```diff
@@ -215,109 +199,98 @@ function erd(self, up, down)
                 up = up(1);
             end
             
-            self.schema.conn.erd({self.fullTableName},up,down)
+            d = dj.ERD(self);
+            n = length(d.nodes);
+            while down>0 || up>0
+                if down>0
+                    down = down - 1;
+                    d.down
+                end
+                if up>0
+                    up = up - 1;
+                    d.up
+                end
+                if length(d.nodes)==n
+                    break
+                end
+                n = length(d.nodes);
+            end
+            d.draw
         end
         
         
         function str = re(self)
-            % dj.Table/re - "reverse engineer" the table declaration.
+            % alias for self.re
+            warning 'dj.Table/re is deprecated: Use dj.Table/describe instead'
```

Comment:
Is there a good reason to remove it in the near future?  I think we will let it linger around for a bit longer and then add the message once no one uses it anymore.